citeproc/model.py

```diff
@@ -767,6 +770,8 @@ def _term(self, item):
         if form == 'long':
             form = None
         term = self.get_term(self.get('term'), form)
+        if term is None:
+            return
```

Comment:
I think we should start using walrus assignment, := introduced in 3.8 -- should make code more readable. Here (and in other places) I think it could be smth like

```suggestion
        if not (term := self.get_term(self.get('term'), form)):
            return
```